README.md

```diff
@@ -36,52 +41,46 @@ development stage (no stable release available).
 
 To compile LibrePCB, you need the following software components:
 - g++ >= 4.8, MinGW >= 4.8, or Clang >= 3.3 (C++11 support is required)
-- Qt >= 5.2 (http://www.qt.io/download-open-source/)
-- `libglu1-mesa-dev` (`sudo apt-get install libglu1-mesa-dev`)
-- `zlib` / `zlib-dev` (http://www.zlib.net/)
-- `OpenSSL` (https://www.openssl.org/)
+- [Qt](http://www.qt.io/download-open-source/) >= 5.2
+- [zlib](http://www.zlib.net/)
+- [OpenSSL](https://www.openssl.org/)
 
-#### Installation on Ubuntu 14.04 and later
+#### Installation on Debian/Ubuntu/Mint
 
 ```bash
-sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator libglu1-mesa-dev zlib1g zlib1g-dev openssl
+sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator \
+libglu1-mesa-dev zlib1g zlib1g-dev openssl
 ```
 
-#### Installation on ArchLinux
-
-You can install [librepcb-git](https://aur.archlinux.org/packages/librepcb-git/) from the AUR.
```

Comment:
zlib is required for both compile-time and runtime, so headers and shared libraries are needed. OpenSSL is only required as shared library at runtime.

Btw, `qt5-tools` is only required for building, it's no runtime dependency. But `qt-base` is both, a runtime and compile-time dependency, see [bitcoin-qt](https://www.archlinux.org/packages/community/x86_64/bitcoin-qt/) for example.

And why do we need `qconf`?